frontend/src/components/SharedEventsPage/SharedEventsPage.jsx

```diff
@@ -1,10 +1,31 @@
 import "./SharedEventsPage.css"
+import { useState, useEffect } from "react";
+import { fetchInvitations, respondToInvitation } from "../../utils/utils";
 
 function SharedEventsPage () {
+    const [invitations, setInvitations] = useState([]);
+
+    useEffect(() => {
+        fetchInvitations(setInvitations);
```

Comment:
these fetch functions are good places to have loading states

Reply:
should the fetchInvitations have a try/catch block on it? You should have some kind of error resolution here for the UI.